mmdet3d/datasets/pipelines/loading.py

```diff
@@ -238,6 +240,104 @@ def __repr__(self):
         return f'{self.__class__.__name__}(sweeps_num={self.sweeps_num})'
 
 
+@PIPELINES.register_module()
+class LoadCalibPoseTime(object):
+    """load calib poses and times for SemanticKitti dataset."""
+
+    def __init__(self, use_calib=True, use_pose=True, use_time=False):
+
+        # self.data_root = data_root
```

Comment:
Please remove unused comments.

Reply:
The unused comments in all files have been removed.